setup.py

```diff
@@ -13,7 +13,7 @@
 
 if 'install' in sys.argv:
 	# naively guess where the autostart .desktop file should be installed
-	if '--prefix' in sys.argv or '--home' in sys.argv:
+	if len([x.startswith('--prefix=') for x in sys.argv]) > 0 or '--home' in sys.argv:
```

Comment:
This should probably just use `any(x.startswith('--prefix') for x in sys.argv)`

Reply:
Sure. Added a commit that does that.